lib/jsonValidator/jsonValidator.ts

```diff
@@ -0,0 +1,31 @@
+import { Schema, Validator } from "jsonschema";
+import { NextApiRequest, NextApiResponse } from "next";
+
+export type validateOptions = {
+  jsonKey: string;
+};
+
+const validate = (
```

Comment:
I still need unit tests for this. You wrote unit tests for the template handler but this doesn't provide adequate coverage for this file. We need unit tests specifically for the validate middleware